run_test.sh

```diff
@@ -1,8 +1,42 @@
 # Script run on Travis CI
 # Runs the Jetpack extension tests
```

Comment:
This comment should be updated to reflect that it is the preferred way to run tests locally as well.